Classes/YLTableViewDataSource.h

```diff
@@ -16,8 +16,14 @@ NS_ASSUME_NONNULL_BEGIN
 //! If one of model's properties has changed (but it's still the same object), this will reload the visible cell for that model. Useful whenever something mutates the state of a model (i.e. image loading)
 - (void)reloadVisibleCellForModel:(id)model inTableView:(UITableView *)tableView;
 
+//! If the cells implement the YLTableViewCellEstimatedRowHeight protocol, that height will be used. Otherwise, the tableView can use an alternate height specified in this method.
```

Comment:
I think this method doc could use a lot more detail, so maybe move to using the /*! */. Things I'd like to see better explained:

- This is only called in the case that the cell does not conform to YLTableViewCellEstimatedRowHeight
- If this method is not overridden, and the cell in question does not conform to YLTableViewCellEstimatedRowHeight, we return UITableViewAutomaticDimension for the estimate row height